sonic-psud/scripts/psud

```diff
@@ -244,9 +244,12 @@ class PsuChassisInfo(logger.Logger):
     def _set_psu_master_led(self, master_status):
         try:
             try:
-                from sonic_platform.psu import Psu
+                if os.environ["PSUD_UNIT_TESTING"] == "1":
+                    from tests.mock_platform import MockPsu as Psu
+                else:
+                    from sonic_platform.psu import Psu
             except ImportError as e:
-                from tests.mock_platform import MockPsu as Psu
+                raise ImportError (str(e) + " - required module not found")
```

Comment:
@mprabhu-nokia: Is there a reason we can't do these imports at the top of the file with the others?

Also, I don't think we need to import DeviceBase, as line 254 should be able to reference `Psu.STATUS_LED_COLOR_GREEN` and `Psu.STATUS_LED_COLOR_RED` becuase Psu inherits from PsuBase which in turn inherits from DeviceBase. Can you please experiment with this and see if I'm correct?

Reply:
@jleveque: Yes, you are right. We can refer to inherited class itself. Cleaned up in psud and in test_psud.py as well. Thanks!

I imported Psu object in here because I was not sure if it would impact fixed-platforms in anyway. Setting of the master-led status in limited to modular chassis. For modular chassis, there was no impact. Please suggest if there would be any impact.

Also, while running on the platform, I got a key-error for "PSUD_UNIT_TESTING". Should this be changed to also check if the env variable exists like below:
```
if "PSUD_UNIT_TESTING" in os.environ and os.environ["PSUD_UNIT_TESTING"] == "1":
```

Reply:
@mprabhu-nokia You can use the next condition:
` if os.getenv("PSUD_UNIT_TESTING") == "1":`

Reply:
@bratashX Thanks!
@jleveque - If this is a genuine issue, can I use this PR to clean the env-variable check in thermalctld and chassisd?

Reply:
@mprabhu-nokia: What do you mean, "clean the env-variable check in thermalctld and chassisd"?

Reply:
Looks like @bratashX  already fixed it as part of f63ec30521d690e9bbef76fdbbed52e4af56ec44